test/parallel/test-stream-writable-destroy.js

```diff
@@ -249,6 +276,9 @@ const assert = require('assert');
   write.destroy(expected, common.mustCall(function(err) {
```

Comment:
destroy with error is now able to override the no error state from the destroy without error above.